CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 ##### Breaking Changes :mega:
 
+- Added `Color.toCssHexString` for getting the CSS hex string equivalent for a color.
```

Comment:
This should go under the `Additions` section instead of the `Breaking Changes` section